package.json

```
@@ -12,6 +12,7 @@
 		"@evidence-dev/evidence": "link:packages/evidence",
 		"@evidence-dev/faker-datasource": "link:packages/datasources/faker",
 		"@evidence-dev/icons": "link:packages/ui/icons",
+		"@evidence-dev/javascript": "link:packages/datasources/javascript",
```

```suggestion
		"@evidence-dev/source-javascript": "link:packages/datasources/javascript",
```

Then this should be good to go